### ISSUE TITLE
Consider CDB-X prototypes as part of some experiments or potential easier source of data, if useful

### ISSUE THREAD

**Issue by serich** [opengeospatial/OGC-ISG-Sprint-Sep-2020#8]
Per @jerstlouis Kickoff presentation, we need to coordinate which CDB version. We might have to stick with whatever is in the San Diego dataset.

Any newer versions that we don't have a chance to test can certainly be discussed under the Future Recommendations section of the Sprint Report.

I'll just assign this to Leonard for starters. @DrX3D feel free to reassign it as needed.

cc:  @holly-black

**Comment by vwhisker** [comment]
As it stands, I dont think there's any data formatted as "CDB-X" from the CDB2/CDB-X concept group.  The available SanDiego CDB is version 1.1, I believe.

**Comment by holly-black** [comment]
Yes, you will not find a full CDB in CDB X.  The concepts for CDB X are still in develop with only pieces and parts of test data in the works.

**Comment by jerstlouis** [comment]
@vwhisker  @holly-black  We should have a first prototype CDBX set of GeoPackages, including tiled imagery, vector and glTF 3D models, produced from the San Diego CDB before the sprint starts.

Even if that doesn't end up being the definitive CDB X, that might potentially be an interesting source of data easier to approach for some ISG participants than CDB 1.x, or might be useful for some of Scenario 1.

@serich I think this issue would be better stated as "Consider CDB-X prototypes as part of some experiments or potential easier source of data, if useful?".
